### PR TITLE
Account prompt

### PR DESCRIPTION
**Pull request by hackaugusto**

### CODE REVIEW

File: raiden/utils/typing.py [hunk 1]
```diff
@@ -93,6 +93,12 @@
 T_PaymentAmount = int
 PaymentAmount = NewType('PaymentAmount', T_PaymentAmount)
 
+T_PrivateKey = bytes
+PrivateKey = NewType('PrivateKey', T_PrivateKey)
+
+T_PublicKey = bytes
+PublicKey = NewType('PrivateKey', T_PublicKey)
```
Comment (pirapira) [inline raiden/utils/typing.py]: Typo.  'PrivateKey' should be 'PublicKey'.

File: raiden/ui/prompt.py [hunk 2]
```diff
@@ -1,89 +1,85 @@
...
+            password = getpass.getpass(
+                f'Enter the password to unlock {address_hex}: ',
+            )
+            return account_manager.get_privkey(address_hex, password)
         except ValueError:
-            # ValueError exception raised if the password is incorrect
-            click.secho(
-                f'Incorrect password for {address_hex} in file. Aborting ...',
-                fg='red',
+            print(
```
Comment (rakanalh) [inline raiden/ui/prompt.py]: There's no breaking out of the loop here... this could go on forever

Comment (hackaugusto) [reply raiden/ui/prompt.py]: Oh, I assumed the message was good enough, I didn't think about the user being stuck and not knowing how to quit. Should I revert it?

Comment (hackaugusto) [reply raiden/ui/prompt.py]: done

**Pull request merged**